deno.jsonc

```diff
@@ -6,7 +6,8 @@
     "all": "deno task udd && deno lint && deno fmt",
     "test": "deno test --unstable",
     "start": "deno run -A --unstable main.ts",
-    "ngrok": "ngrok http 8080"
+    "ngrok": "ngrok http 8080",
+    "cf:deploy": "deno run -A cloudflare/deploy.ts"
```

Comment:
TODO: Add task [`cf:serve`](https://denoflare.dev/guides/serve).

Reply:
Done in <https://github.com/acmcsufoss/lc-dailies/pull/15/commits/bf84e1781195c2990d64c6f940c4fd23489284e8>.